packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -448,7 +448,20 @@ class TypeMatcher<T> {
 /// updating the underlying element (i.e., by calling [Element.update] with the
 /// new widget). Otherwise, the old element is removed from the tree, the new
 /// widget is inflated into an element, and the new element is inserted into the
-/// tree.
+/// tree. Basically any time you are building a list of widgets from a list of
+/// data, and the list might change, you should set a key to the children, so
+/// that we keep the subtrees tied to the data. Otherwise, we will not know
```

Comment:
I recommend continuing to use "you" instead of "we" for consistency. "...so that you keep the subtrees tied to the data."
"otherwise, you will not know.."

Reply:
Otherwise, if anything [is there a more precise word for that? any widget? something else?] is stateful, the state could spill...

Reply:
I was actually thinking to change 'We' with 'Flutter' in this case since it is talking about the work that the framework will be doing.

What do you think?

Reply:
Sounds good